pkgs/by-name/qw/qwerty-fr/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchurl
+, lib
+, unzip
+,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qwerty-fr";
+  version = "0.7.3";
+
+  src = fetchurl {
+    url = "https://github.com/qwerty-fr/qwerty-fr/releases/download/v${version}/qwerty-fr_${version}_linux.zip";
```

Comment:
Use the [`stdenv.mkDerivation (finalAttrs: { })`](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes) pattern to access `version` in the src fetcher, instead of a recursive attrset